src/models/outgoing_edges.rs

```diff
@@ -23,22 +23,43 @@ pub(crate) struct Edges {
 
 // Captures an entry from the `edges.toml` file.
 #[derive(Deserialize, Debug, Clone, Hash, PartialEq, Eq, Default, Getters, Builder)]
+#[pyclass]
 pub struct OutgoingEdges {
+  /// The source rule or group of rules
   #[get = "pub with_prefix"]
-  from: String,
+  #[serde(alias = "from")]
```

Comment:
Interesting. This is due to `from` being a keyword in Python, right?

Reply:
Yes. :O